### ISSUE TITLE
Fails to parse trailing comment after JS object literal open curly brace

### ISSUE THREAD

**Issue by christopherthielen** [decaffeinate/decaffeinate#61]
Failing test case:
```
RestangularProvider.setRestangularFields({ # comment
  route: "restangularRoute"
})
```
error: 
```
Error: Syntax error on line 6, column 3: unexpected 'r' (\u0072)
3 :   route: "restangularRoute"
4 : # Fails
5 : RestangularProvider.setRestangularFields({ # comment
6 :   route: "restangularRoute"
^ :   ^
7 : })
```
I suspect this might be a Redux problem

---

Side note: Switching the JS object literal to a coffee literal, or moving the comment to the next line works:
```
# These work fine
RestangularProvider.setRestangularFields # comment
  route: "restangularRoute"

RestangularProvider.setRestangularFields({
  # comment
  route: "restangularRoute" # comment
})
```

**Comment by eventualbuddha** [comment]
Yep, CSR [fails to parse this one](https://michaelficarra.github.io/CoffeeScriptRedux/#try:RestangularProvider.setRestangularFields(%7B%20%23%20comment%0A%20%20route%3A%20%22restangularRoute%22%0A%7D)). This may be fixed by #65.

**Comment by eventualbuddha** [comment]
Fixed in the latest version ([repl](http://decaffeinate.github.io/decaffeinate/repl/#?evaluate=false&code=RestangularProvider.setRestangularFields(%7B%20%23%20comment%0A%20%20route%3A%20%22restangularRoute%22%0A%7D))).
